auslib/web/views/client.py

```diff
@@ -10,8 +10,10 @@
 
 
 class ClientRequestView(MethodView):
-
     def __init__(self, *args, **kwargs):
+        # By default, we want a cache that can be shared across requests from different users ("public")
+        # and a maximum age of 60 seconds, to keep our TTL low.
+        self.cacheControl = app.config.get("CACHE_CONTROL", "public,max-age=60")
```

Comment:
I think it's probably OK to trust it. In production, Ops people would be overwriting it, and I think we can trust that they know what they're doing :). I'm not sure how we'd verify it either unless there's some library that exists. I suppose we could only allow specific parameters (like max-age value) to be passed, but I think it's better to allow this to be fully overridden.